packages/jaeger-ui/src/components/TracePage/TraceStatistics/index.tsx

```diff
@@ -479,12 +439,32 @@ export default class TraceStatistics extends Component<Props, State> {
         {this.state.showPopup ? (
           <PopupSQL closePopup={this.togglePopup} popupContent={this.state.popupContent} />
         ) : null}
-        <table className="test1893">
+        {/* <table className="test1893">
           <tbody className="DetailTraceTableTbody--TraceStatistics">
             {this.renderTableHead()}
             {this.renderTableData()}
           </tbody>
-        </table>
+        </table> */}
+        <Table
+            className="span-table span-view-table"
+            columns={columns}
+            dataSource={isNoDetail}
+            pagination={{
+              total: isNoDetail.length,
+              pageSizeOptions: ['10', '20', '50', '100'],
+              showSizeChanger: true,
+              showQuickJumper: true,
+            }}
+            rowClassName={(record,index)=>{
+              if(record.type === 'undefined'){
+                return "undefClass"
+              }
```

Comment:
Is this needed? Can we simplify the code by removing this?

Reply:
Yes this returns a styling to the row whose type is undefined